<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GambleManage</class>
 <widget class="QWidget" name="GambleManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加奖品</string>
   </property>
  </widget>
  <widget class="QTableView" name="histView">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>171</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除卡池</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="infoEdt">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>181</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>上线</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
